services/src/main/java/org/keycloak/protocol/saml/profile/ecp/authenticator/HttpBasicAuthenticator.java

```diff
@@ -52,17 +53,24 @@ public String getDisplayType() {
 
     @Override
     public String getReferenceCategory() {
-        return null;
+        return "basic";
     }
 
```

Comment:
@stianst If this existing class is usable beyond saml ecp protocol I suggest we move it to *authentication/authenticators/browser*